projects/RabbitMQ.Client/client/impl/Connection.cs

```diff
@@ -127,6 +128,18 @@ public Connection(IConnectionFactory factory, bool insist, IFrameHandler frameHa
             }
         }
 
+        public Connection(IConnectionFactory factory, bool insist, IFrameHandler frameHandler, ArrayPool<byte> memoryPool,
+                string clientProvidedName = null)
+            : this(factory, insist, frameHandler, clientProvidedName)
+        {
+            _memoryPool = memoryPool;
```

Comment:
Given we keep the factory around what is the benefit of the additional property? It seems to only avoid having to type the small train wreck which seems neglects le given the pool is anyway designed to be managed by the factory

Reply:
`factory` is `IConnectionFactory` which we can't change in 6.3 (correct?)

Reply:
Yeah correct. I got mislead because I didn't look properly at the verified file